### PR TITLE
Fix for tag and search command options

### PR DESCRIPTION
**Pull request by praneethr86**
- StreamItems is the right reference for 'tag' and 'search' options as well as per Medium API response. Fixed to use that correctly.
- Loading the 'post' object from postMap gives the right data as well.

### CODE REVIEW

File: lib/post.js [hunk 1]
```diff
@@ -56,7 +56,7 @@ function processContent(data, modifier, count) {
 	// get items from json response
 	switch (modifier) {
 		case 'tag':
-			items = data.payload.value;
+			items = data.payload.streamItems;
```
Comment (djadmin) [inline lib/post.js]: Good catch.

**Comment by djadmin** [comment]
Thank you @codePrady for the patch. 

I've left couple comments on the PR, please check them. And I request you to squash all your commits in one.

Thanks again 👍

**Comment by djadmin** [comment]
Merged. Thank you !

**Comment by praneethr86** [comment]
Thanks for the review comments. Will make a note next time.

**Pull request merged**